src/react/hoc/subscription-hoc.tsx

```diff
@@ -17,8 +17,8 @@ import { OperationOption, OptionProps, DataProps } from './types';
 export function withSubscription<
   TProps extends TGraphQLVariables | {} = {},
   TData = {},
-  TGraphQLVariables = {},
-  TChildProps = DataProps<TData, TGraphQLVariables>
+  TGraphQLVariables extends {} = {},
+  TChildProps extends { [x: string]: any; } = DataProps<TData, TGraphQLVariables>
```

Comment:
A possible alternative:

```suggestion
  TChildProps extends Partial<DataProps<TData, TGraphQLVariables>> = DataProps<TData, TGraphQLVariables>
```

or possibly, but suspiciously

```suggestion
  TChildProps extends {} = DataProps<TData, TGraphQLVariables>
```